src/components/tipRecords/AuthorAndDate.vue

```diff
@@ -1,48 +1,101 @@
 <template>
-  <div
-    class="author-and-date"
-    @click.stop
-  >
-    <RouterLink
-      :to="{
-        name: 'user-profile',
-        params: {
-          address,
-        },
-      }"
+  <div class="wrapper">
+    <UserHoverCard
+      :address="address"
+      :hover="hover"
+      :show="show"
+      :is-active="isActive"
+      :mouseover="mouseover"
+      :mouseleave="mouseleave"
+      :mousestay="mousestay"
+    />
```

Comment:
Moving transition out from UserHoverCard, and using `transition-delay` it should be possible to simplify it to:
```suggestion
    <Transition ...>
      <UserCard
        v-if="hover"
        :address="address"
        @mouseover="mouseover"
        @mouseleave="mouseleave"
      />
    </Transition>
```